watchdog bundle implementation

u7702045 opened the pull request:
realm

u7702045 committed:
watchdog bundle implementation

u7702045 committed:
remove unnecessary comment

u7702045 committed:
add travis configuration

u7702045 committed:
add GET test case

imZack commented on bundle.json:
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "watchdog monitor",
```
name shoud not have space

u7702045 committed:
[USERS]

	* fix bundle name with '-'
	* remove hardcode path "/home/moxa" instead of user define path

imZack commented on watchdog.py:
```diff
@@ -0,0 +1,82 @@
+                fd.write("\tstop program = \"/usr/bin/killall -9 %s\"\n"
+                         % process["process"])
+
+            fd.closed
```
fd.closed?

u7702045 committed:
* add branket after fd.close

u7702045 committed:
* fix fd.close function call

u7702045 commented:
ya.... but why no error when running watchdog.py?? will fix it!!
